### PR TITLE
Set job status to processed after completion

### LINKED ISSUE #35: Implement state change in ToolingJob::Process
@ErikSchierboom: Something to get you into the Dynamodb codebase.

Once the processing is done in `ToolingJob::Process`, we want to update the status of the item to be processed.

This code from tooling invoker will probably be useful to you.

```ruby
      client.update_item(
        table_name: table_name,
        key: { id: job_id },
        expression_attribute_names: {
          "#JS": "job_status",
          "#LU": "locked_until"
        },
        expression_attribute_values: {
          ":js": "locked",
          ":lu": Time.now.to_i + 15
        },
        update_expression: "SET #JS = :js, #LU = :lu",
        return_values: "ALL_NEW"
      )
```

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes #35

### CODE REVIEW

File: app/commands/tooling_job/process.rb [hunk 1]
```diff
@@ -40,6 +40,20 @@ def call
...
+        expression_attribute_names: {
+          "#JS": "job_status",
+          "#LU": "locked_until"
+        },
+        expression_attribute_values: {
+          ":js": "processed",
+          ":lu": nil
+        },
+        update_expression: "SET #JS = :js, #LU = :lu"
+      )
```
Comment (ErikSchierboom) [inline app/commands/tooling_job/process.rb:54]: The [example code you linked to](https://github.com/exercism/v3-website/issues/35#issue-674628338) used `return_values: "ALL_NEW"`. As we're not actually using the returned values from the `update_item` method, I've omitted this line for optimization purposes (see [the docs](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html)).

File: app/commands/tooling_job/process.rb [hunk 2]
```diff
@@ -40,6 +40,20 @@ def call
...
+      client.update_item(
+        table_name: Exercism.config.dynamodb_tooling_jobs_table,
+        key: { id: id },
+        expression_attribute_names: {
+          "#JS": "job_status",
+          "#LU": "locked_until"
+        },
+        expression_attribute_values: {
+          ":js": "processed",
+          ":lu": nil
```
Comment (ErikSchierboom) [inline app/commands/tooling_job/process.rb]: I've set the `locked_until` property to `nil`. Is that the right thing to do, or do you want to leave it unchanged?

Comment (iHiD) [reply app/commands/tooling_job/process.rb]:
```suggestion
```

File: app/commands/tooling_job/process.rb (hunk 1 above)
Comment (iHiD) [reply app/commands/tooling_job/process.rb:54]: Great :)

File: test/test_helper.rb [hunk 3]
```diff
@@ -71,6 +71,15 @@ def write_to_dynamodb(table_name, item)
...
   end
 
+  def read_from_dynamodb(table_name, key, attributes)
+    client = ExercismConfig::SetupDynamoDBClient.()
+    client.get_item(
+      table_name: table_name,
+      key: key,
+      attributes_to_get: attributes
+    ).item
+  end
```
Comment (iHiD) [inline test/test_helper.rb:81]: Nice.

**Review by iHiD — APPROVED** [review]
This is great. Thanks :) 

I'll approve this and you can merge it once you've removed the locked stuff

**Pull request merged**